File: js/location.js
class Location {
  constructor(longitude, latitude) {
    this._long = longitude;
    this._lat = latitude;
  }

  getLongitude() {
    return this._long;
  }

  getLatitude() {
    return this._lat;
  }

  setLongitude(longitude) {
    this._long = longitude;
  }

  setLatitude(latitude) {
    this._lat = latitude;
  }
}

File: python/location.py

class Location:
    def __init__(self, longitude, latitude):
        self._long = longitude
        self._lat = latitude

    def get_longitude(self):
        return self._long

    def get_latitude(self):
        return self._lat

    def set_longitude(self, longitude):
        self._long = longitude

    def set_latitude(self, latitude):
        self._lat = latitude
    
    